tests/integ/test_dataframe.py

```diff
@@ -1956,6 +1956,50 @@ def test_case_insensitive_collect(session):
     assert row["p@$$w0rd"] == "test"
     assert row["P@$$W0RD"] == "test"
 
+def test_case_insensitive_local_iterator(session):
+    df = session.create_dataframe(
+        [["Gordon", 153]], schema=["firstname", "matches_won"]
+    )
+    df_quote = session.create_dataframe(
+        [["Gordon", 153]], schema=["'quotedName'", "quoted-won"]
+    )
+
+    # tests for sync collect
+    row = list(df.to_local_iterator(case_sensitive=False))[0]
```

Comment:
NIT: I prefer using `next` instead of `list(...)[0]`

Reply:
Done